cmd/snap/cmd_snap_op.go

```diff
@@ -350,13 +350,27 @@ func maybeWithSudoSecurePath() bool {
 	return release.DistroLike("fedora", "opensuse", "debian")
 }
 
+func hasWaitTasks(chg *client.Change) bool {
+	for _, t := range chg.Tasks {
+		if t.Status == "Wait" {
+			return true
+		}
+	}
+	return false
+}
+
 // show what has been done
-func showDone(cli *client.Client, names []string, op string, opts *client.SnapOptions, esc *escapes) error {
+func showDone(cli *client.Client, chg *client.Change, names []string, op string, opts *client.SnapOptions, esc *escapes) error {
 	snaps, err := cli.List(names, nil)
 	if err != nil {
 		return err
 	}
 
+	if hasWaitTasks(chg) {
+		fmt.Fprintf(Stdout, i18n.G("Change %v waiting to be completed\n"), chg.ID)
```

Comment:
waiting on external action? otherwise is a bit unclear for the user what to expect